CONTRIBUTING.md

```diff
@@ -3,196 +3,113 @@
-bash Miniconda3-latest-MacOSX-x86_64.sh
-
-# During the installation, conda edits your .bashrc to add its paths
-# Either re-open the terminal, or
-source ~/.bashrc  # Linux
-source ~/.bash_profile  # MAC
-
-# Install conda-build which is needed to correctly setup a development environment:
-conda install -n base -c conda-forge conda-build
-```
-
-**Note:** the following steps guide you through the manual process of creating a development environment.
-For convenience, there is a `ilastik-meta/ilastik/scripts/devenv.py` script that automates this process.
-Ensure that your current working directory is `ilastik-meta` and run `python ilastik/scripts/devenv.py create -n idev`.
-
-#### Manual Creation of development environment
-Create the ilastik development environment (we assume that the dev-environment will have the name _idev_, but you can, of course choose a name to your liking):
-```bash
-conda create --name idev -c ilastik-forge -c conda-forge ilastik-dependencies-no-solvers
-```
-
-In order to run your own code, you need to install the local ilastik packages with `conda develop`.
-
-```bash
-# from you local ilastik-meta folder
-git submodule foreach 'conda develop .'
-```
-
-Now it might be the time to test whether the installation was successful:
-```bash
-# activate your conda environment
-conda activate idev
-
-# check whether importing ilastik repos works
-python -c "import ilastik; import volumina"
-```
-
-If there are no errors here, chances are high the development environment was set up correctly.
-If the above line already works, you can start ilastik by:
-
-```bash
-# in ~/source/ilastik-meta/ilastik
-python ilastik.py
-```
-
-More information on conda, and building the ilastik packages yourself:
- * [miniconda](https://conda.io/miniconda.html)
- * [conda user guide](https://conda.io/docs/user-guide/index.html)
- * [conda build docs](https://conda.io/docs/commands/build/conda-build.html)
- * [On building ilastik packages with conda-build](https://github.com/ilastik/ilastik-publish-packages)
-
-## Coding
-
-In order to follow the [GitHub Flow](https://guides.github.com/introduction/flow/), please always start from a current master (see the above section on how to sync your forks) and create a feature branch that you will push to your own fork.
+## Create a Development Environment
+
+1. Download and install [GitHub CLI](https://cli.github.com/).
+
+1. Fork and clone repositories:
+    ```
+    gh repo fork --clone=true --remote=true ilastik/volumina
+    gh repo fork --clone=true --remote=true ilastik/ilastik
+    ```
+
+    If you already forked repositories before, just clone them:
+    ```
+    gh repo clone volumina
+    gh repo clone ilastik
+    ```
+
+1. Download and install _the latest 64-bit_ [miniconda](https://docs.conda.io/en/latest/miniconda.html).
+
+1. Install [mamba](https://github.com/mamba-org/mamba) and [conda-develop](https://docs.conda.io/projects/conda-build/en/latest/resources/commands/conda-develop.html):
```

Comment:
I'm not buying into mamba just yet. Our whole distribution stack still uses conda and in order to get the same environments, I'd still prefer to use conda. See also https://github.com/mamba-org/mamba/issues/525

Reply:
for `ilastik-dependencies-no-solvers` we'd get the following diff:

<details>

```
Common Packages (with diff)
---------------------------
  importlib-metadata
    build_number: 0 -> 1
    build_string: py37hc8dfbb8_0 -> py_1
    dist_name: importlib-metadata-2.0.0-py37hc8dfbb8_0 -> importlib-metadata-2.0.0-py_1
    platform: linux-64 -> noarch
  jsonschema
    build_number: 1 -> 2
    build_string: py37hc8dfbb8_1 -> py_2
    dist_name: jsonschema-3.2.0-py37hc8dfbb8_1 -> jsonschema-3.2.0-py_2
    platform: linux-64 -> noarch
  libblas
    build_number: 1 -> 17
    build_string: 1_h86c2bf4_netlib -> 17_openblas
    dist_name: libblas-3.9.0-1_h86c2bf4_netlib -> libblas-3.8.0-17_openblas
    version: 3.9.0 -> 3.8.0
  libcblas
    build_number: 1 -> 17
    build_string: 1_h86c2bf4_netlib -> 17_openblas
    dist_name: libcblas-3.9.0-1_h86c2bf4_netlib -> libcblas-3.8.0-17_openblas
    version: 3.9.0 -> 3.8.0
  liblapack
    build_number: 1 -> 17
    build_string: 1_h86c2bf4_netlib -> 17_openblas
    dist_name: liblapack-3.9.0-1_h86c2bf4_netlib -> liblapack-3.8.0-17_openblas
    version: 3.9.0 -> 3.8.0
  libpq
    build_string: h1281834_2 -> h5513abc_2
    dist_name: libpq-12.3-h1281834_2 -> libpq-12.3-h5513abc_2
  nose
    build_number: 1004 -> 1006
    build_string: py37hc8dfbb8_1004 -> py_1006
    dist_name: nose-1.3.7-py37hc8dfbb8_1004 -> nose-1.3.7-py_1006
    platform: linux-64 -> noarch
  pyopenssl
    build_number: 0 -> 1
    build_string: py37_0 -> py_1
    dist_name: pyopenssl-19.1.0-py37_0 -> pyopenssl-19.1.0-py_1
    platform: linux-64 -> noarch
  python
    build_string: h425cb1d_1_cpython -> h6f2ec95_1_cpython
    dist_name: python-3.7.8-h425cb1d_1_cpython -> python-3.7.8-h6f2ec95_1_cpython
  scikit-learn
    build_number: 0 -> 1
    build_string: py37_blas_openblashebff5e3_0 -> py37ha8026db_1
    dist_name: scikit-learn-0.20.4-py37_blas_openblashebff5e3_0 -> scikit-learn-0.20.3-py37ha8026db_1
    version: 0.20.4 -> 0.20.3

Packages Only In /tmp/mamba-env.json:
--------------------------
  blas
  libgfortran
  openblas

Packages Only In /tmp/conda-env.json:
--------------------------
  libopenblas
```

</details>

Reply:
Thanks for pointing that, I considered mamba to be a drop-in replacement before.